src/cascadia/UnitTests_SettingsModel/SerializationTests.cpp

```diff
@@ -493,7 +497,7 @@ namespace SettingsModelUnitTests
                 }
             ],
             "actions": [
-                { "command": { "action": "sendInput", "input": "VT Griese Mode" }, "keys": "ctrl+k" }
+                { "command": { "action": "sendInput", "input": "VT Griese Mode" }, "id" : "User.sendInput.15093368865568800249", "keys": "ctrl+k" }

```

Comment:
Maybe we can truncate the hash down to, i dunno, 32 bits? and write it in hex. That way, this will show up as `User.sendInput.E02B3DF9`. Less unwieldy. Sure, there's a chance for collision, but it's pretty unlikely for the amount of commands a user has.